recipe/meta.yaml

```diff
@@ -38,21 +38,22 @@ requirements:
     # Correlations
     - phik >=0.11.1,<0.13
     # Examples
-    - requests >=2.24.0,<2.29
+    - requests >=2.24.0,<3
     # Progress bar
-    - tqdm >=4.48.2,<4.65
+    - tqdm >=4.48.2,<5
     - seaborn >=0.10.1,<0.13
-    - multimethod >=1.4,<1.10
+    - multimethod >=1.4,<2
     # metrics
-    - statsmodels >=0.13.2,<0.14
-    - typeguard >=2.13.2,<2.14
+    - statsmodels >=0.13.2,<1
+    # type checking
+    - typeguard >=2.13.2,<3
     - imagehash ==4.3.1
-    - pillow
+    - wordcloud >=1.8.2.2
+    - dacite >=1.6
   run_constrained:
     - jupyter-client >=5.3.4
     - jupyter-core >=4.6.3
     - tangled-up-in-unicode 0.2.0
-    - ipywidgets >=7.5.1
```

Comment:
`ipywidgets >=7.5.1` is still required here https://github.com/ydataai/ydata-profiling/blob/116fb8257ff49b0071ac321779739643650fe59e/setup.py#L38

Reply:
@Arishamays1 can you add this back please?

Reply:
Its been added!

Reply:
You added it in the runtime dependencies. It should be in the `run_constrained` block.

Reply:
Ahh thanks for letting me know. Updated